admin/class-yoast-network-admin.php

```diff
@@ -143,12 +159,13 @@ public function handle_update_options_request() {
 	public function handle_restore_site_request() {
 		$this->verify_request( 'wpseo-network-restore', 'restore_site_nonce' );
 
-		$option_group = 'wpseo_ms';
+		// TODO: replace hardcoded string with network_admin_options_service->option_name.
+		$option = 'wpseo_network_admin_options';
```

Comment:
Note: line 180 also need to be changed to use this instance.
```suggestion
        /**
         * Indicates this is a Network_Admin_Options_Service instance.
         *
         * @var Network_Admin_Options_Service $network_admin_options_service
         */
        $network_admin_options_service = YoastSEO()->classes->get( Network_Admin_Options_Service::class );

		$option = $network_admin_options_service->option_name;
```

Reply:
Eep, good catch 🙏